postcss.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    parser: '',
+    plugins: [
+        require("autoprefixer")({
+            browsers: ['last 2 versions','ie 11','> 5%','Firefox > 35','Chrome > 35'],
```

Comment:
Just noticed that it doesn't mention edge. We haven't updated this config since last year. May be we can fix it in a separate PR later on.

Reply:
Edge should be captured by `last 2 versions`.

https://github.com/browserslist/browserslist#full-list